service/resource/dnsrecord/resource.go

```diff
@@ -0,0 +1,351 @@
+package dnsrecord
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/Azure/azure-sdk-for-go/arm/dns"
+	"github.com/Azure/go-autorest/autorest/to"
+	"github.com/giantswarm/azuretpr"
+	"github.com/giantswarm/microerror"
+	"github.com/giantswarm/micrologger"
+	"github.com/giantswarm/operatorkit/framework"
+
+	"github.com/giantswarm/azure-operator/client"
+	"github.com/giantswarm/azure-operator/service/key"
+)
+
+const (
+	// Name is the identifier of the resource.
+	Name = "resourcegroup"
```

Comment:
The package name of the resource is `dnsrecord`, so the resource name should be aligned with that IMO.

Reply:
Yes, copy/paste. Thanks!

Reply:
Done.